app/views/incomes/confirm_save_Income_new.scala.html

```diff
@@ -1,87 +0,0 @@
-@*
- * Copyright 2018 HM Revenue & Customs
```

Comment:
Is it intentionally we have deleted the new page and did changes on an old page?